bicycleparameters/main.py

```diff
@@ -726,6 +728,271 @@ def com_symbol(ax, center, radius, color='b'):
+                fig1.add_trace(go.Scatter(x=[x_center,x_center],y=[y_center-R,y_center],mode='lines',line_color="black", showlegend = False,hoverinfo='none'))
+                fig1.add_trace(go.Scatter(x=xc4,y=yc4,mode='lines',line_color="black", showlegend = False,hoverinfo='none'))
+                fig1.add_trace(go.Scatter(x=[x_center,x_center],y=[y_center,y_center],mode='lines',line_color="black",hovertemplate="%{x:.3f}<br>%{y:.3f}",name='COM',showlegend = False))
+                return fig1
+
+            # radius of the CoM symbol
+            sRad = 0.03
+            # front wheel CoM
+            x_com_Wf=par['w']
+            y_com_Wf=par['rF']
+
+            fig1=com_symbol(sRad,x_com_Wf,y_com_Wf)
+            fig1.add_annotation(text=part,
+                  xref='x', yref='y',
+                  x=x_com_Wf+0.055, y=y_com_Wf+0.055,showarrow=False,font=dict(size=15))
+
+            # rear wheel CoM
+            fig1=com_symbol(sRad,0.,par['rR'])
+            fig1.add_annotation(text="R",
+                  xref='x', yref='y',
+                  x=0.055, y=par['rR']+0.055,showarrow=False,font=dict(size=15))
+
+            for j, part in enumerate([x for x in parts
+                                      if x not in 'RFD']):
+                xcom = par['x' + part]
+                zcom = par['z' + part]
+                fig1=com_symbol(sRad,xcom,-zcom)
+                fig1.add_annotation(text=part,
+                  xref='x', yref='y',
+                  x=xcom+0.055, y=-zcom+0.055,showarrow=False,font=dict(size=15))
+
+            if 'H' not in parts:
+                fig1 = com_symbol(sRad,par['xH'], -par['zH'])
+                fig1.add_annotation(text="H",
+                  xref='x', yref='y',
+                  x=par['xH']+0.055, y=-par['zH']+0.055,showarrow=False,font=dict(size=15))
+
+
+        # if there is a rider on the bike, make a simple stick figure
+        top_of_head = 0.0
+        if self.human:
+            human = self.human
+            mpar = self.parameters['Measured']
+            bpar = self.parameters['Benchmark']
+            # K2: lower leg, tip of foot to knee
+            start = rider.yeadon_vec_to_bicycle_vec(human.K2.end_pos, mpar,
+                                                    bpar)
+            end = rider.yeadon_vec_to_bicycle_vec(human.K2.pos, mpar, bpar)
+            fig1.add_trace(go.Scatter(x=[start[0, 0], end[0, 0]],y=[-start[2, 0], -end[2, 0]],mode='lines'))
+
+        #     # K1: upper leg, knee to hip
+            start = rider.yeadon_vec_to_bicycle_vec(human.K2.pos, mpar, bpar)
+            end = rider.yeadon_vec_to_bicycle_vec(human.K1.pos, mpar, bpar)
+
+        fig1.update_layout(title_text='Bicycle geometry',title_x=0.5,
+                   )
+        # fig1.update_layout(yaxis=dict(autorange=True,showgrid=False,ticks='',showticklabels=False))
+        # fig1.update_layout(xaxis=dict(autorange=True,showgrid=False,ticks='',showticklabels=False))
```

Comment:
remove unused code

Reply:
I'm leaving this one in, so you can later on choose which is better to view the plot.

Reply:
Ok, I can test it out.